### ISSUE TITLE
Latest APKs are not on GitHub

### ISSUE THREAD

**Issue by AmyJeanes** [CorvetteCole/PixelWatchFace#70]
As the app is not currently available on the Play Store I tried to install the app from the latest GitHub release but it appears to be an older version than was on the Play Store. Can you please publish a new version with the newest APKs?

**Comment by CorvetteCole** [comment]
Yes, I will. I will also be fixing the play store ASAP. I've been busy with a summer job and I'm about to move to a new place and start another job with a space company so I've been busy!! I will have time after my move and this app is my top priority. I've actually completed preliminary implementation work for a full rewrite and redesign of the phone app to bring it up to my standards and bring some nice features (Material You theming on your watch face anyone?). I apologize, I know I have an obligation to support this watch face especially after people have paid money for it, but I'm one guy and I just haven't had the time.

Don't worry it will be done!

**Comment by davwheat** [comment]
I was actually trying to buy the weather functionality before I realised it said the item was disabled.

> I know I have an obligation to support this watch face especially after people have paid money for it, but I'm one guy and I just haven't had the time.

People paid for the weather functionality, which still works fine. You've done a very fine job so far and I trust you will do in the future too. :)

**Comment by AmyJeanes** [comment]
As a workaround I managed to compile it myself with by own OpenWeatherMap API key, I would share the APKs here but it will include my API key. Thank you for the update and no worries, the app still works great! I hope you can resolve the issue with the Play Store soon

**Comment by Selljo** [comment]
I just had to reset my watch and my phone and I started to get very worried about where this app went. Glad to hear life is keeping you busy and congrats on the job! Looking forward to using this app when it's available, thanks for all the hard work and time and effort!
